changelog/_unreleased/2020-09-17-fix-product-media-position-duplication.md

```diff
@@ -0,0 +1,34 @@
+---
+title:         Fix bug when images are add to a product the position 1 can be assigned multiple times
+issue:         NEXT-9292
+author:        Joshua Behrens
+author_email:  code@joshua-behrens.de
+author_github: JoshuaBehrens
+___
+# Administration
+* Match procedure to assign `product_media.position` in `sw-product-media-form` and `sw-product-detail` so it is always set to prevent a bug where the position 1 is automatically assigned which results in unexpected sorting and selection behaviours
```

Comment:
Please make sure to use one of the defined keywords. 

https://github.com/shopwareBoostDay/platform/blob/master/changelog/readme.md